packages/vm/src/testing/instance.rs

```diff
@@ -17,7 +17,7 @@ use super::storage::MockStorage;
 /// This gas limit is used in integration tests and should be high enough to allow a reasonable
 /// number of contract executions and queries on one instance. For this reason it is significatly
 /// higher than the limit for a single execution that we have in the production setup.
-const DEFAULT_GAS_LIMIT: u64 = 500_000_000; // ~0.5ms
+const DEFAULT_GAS_LIMIT: u64 = 700_000_000; // ~0.7ms
```

Comment:
Oh, is this needed for a single conversion+verification?

Reply:
Unfortunately. I'm not sure what part of it is so expensive though.. I'm inclined to say it's the SHA256 digest since inside of WASM it has to fall back to its software implementation.
But I have to properly bench that

Reply:
374_000_000 of that is the call to `secp256r1_verify`. Hashing is usually very cheap. So it is either the point conversions or the `webauthn_verify_errors` test in which we test 2 queries with one instance. If we create fresh `deps`, maybe we can go back to the original limit?

Reply:
Nevermind, webauthn_verify_errors only does one query

Reply:
Not sure how far it is under 500k now, but I got it there.  
Here we simply saved on memory operations which are also costing gas.